### PR TITLE
Sets source and target versions properly

### PR DESCRIPTION
**Pull request by petterarvidsson**
- We want to generate JAVA 1.7 compatible source code for ease of use

### CODE REVIEW

**Comment by petterarvidsson** [comment]
@nsg Simple fix for getting the right java version for the JAR built.

**Comment by petterarvidsson** [comment]
@nsg Balanced this plugin together with the caves plugin now

**Comment by petterarvidsson** [comment]
ready for review :rocket:

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Generates ore veins
 
 ## Add your own ore
 Other plugins can register ore blocks with this plugin by extending its configuration:
-´´´
+```
```
Comment (nsg) [inline README.md:6]: `'´

File: src/main/java/org/konstructs/ore/Ore.java [hunk 2]
```diff
@@ -29,25 +29,18 @@ public Ore(ActorRef universe, OreConfig config) {
...
-            .addX(randomDistance())
-            .addY(randomDistance())
-            .addZ(randomDistance());
+        Position growthPos = pos.add(Direction.getRandom().getVector().multiply(randomDistance()));
         BoxAround box = new BoxAround(growthPos, radius);
         boxShapeQuery(box);
     }
 
     private void growVein(Position start) {
+        System.out.println("Grow vein: " + start + " " + config.getOre());
```
Comment (nsg) [inline src/main/java/org/konstructs/ore/Ore.java]: Do we like to keep this one in master?

**Review by nsg — APPROVED** [review]
Looks good

**Pull request merged**